packages/now-cli/src/commands/bisect/index.ts

```diff
@@ -0,0 +1,323 @@
+    return 1;
+  }
+
+  if (linkedProject.status === 'error') {
+    return linkedProject.exitCode;
+  }
+
+  const { project, org } = linkedProject;
+  client.currentTeam = org.type === 'team' ? org.id : undefined;
+
+  output.log(`Bisecting project ${chalk.bold(`"${project.name}"`)}`);
+
+  if (!bad) {
+    const answer = await inquirer.prompt({
+      type: 'input',
+      name: 'bad',
+      message: `What's the deployment URL where the bug occurs\n  Leave blank for the latest deployment:`,
+    });
+    bad = answer.bad;
+    output.print('\n');
+  }
+
+  if (bad) {
+    if (!bad.startsWith('https://')) {
+      bad = `https://${bad}`;
+    }
+    const parsed = parse(bad);
+    if (!parsed.hostname) {
+      output.error('Invalid input: no hostname provided');
+      return 1;
+    }
+    bad = parsed.hostname;
+    if (typeof parsed.path === 'string' && parsed.path !== '/') {
+      if (subpath && subpath !== parsed.path) {
+        output.note(
+          `Ignoring subpath ${chalk.bold(
+            parsed.path
+          )} in favor of \`--path\` argument ${chalk.bold(subpath)}`
+        );
+      } else {
+        subpath = parsed.path;
+      }
+    }
+  }
+
+  if (!good) {
+    const answer = await inquirer.prompt({
+      type: 'input',
+      name: 'good',
+      message: `What's a deployment URL where the bug does not occur\n  Leave blank for the oldest deployment:`,
+    });
+    good = answer.good;
+    output.print('\n');
+  }
+
+  if (good) {
+    if (!good.startsWith('https://')) {
+      good = `https://${good}`;
+    }
+    const parsed = parse(good);
+    if (!parsed.hostname) {
+      output.error('Invalid input: no hostname provided');
+      return 1;
+    }
+    good = parsed.hostname;
+    if (
+      typeof parsed.path === 'string' &&
+      parsed.path !== '/' &&
+      subpath &&
+      subpath !== parsed.path
+    ) {
+      output.note(
+        `Ignoring subpath ${chalk.bold(
+          parsed.path
+        )} which does not match ${chalk.bold(subpath)}`
+      );
+    }
+  }
+
+  if (!subpath) {
+    const answer = await inquirer.prompt({
+      type: 'input',
+      name: 'subpath',
+      message: `What's the URL path where the bug occurs:`,
+    });
+    subpath = answer.subpath;
+
+    output.print('\n');
+  }
+
+  // Fetch all the project's "READY" deployments with the pagination API
+  output.spinner('Retrieving deployments…');
+  let next: number | undefined;
+  let deployments: Deployment[] = [];
+  do {
+    const query = new URLSearchParams();
+    query.set('projectId', project.id);
+    query.set('target', 'production');
+    query.set('limit', '100');
+    query.set('state', 'READY');
+    if (next) {
+      query.set('until', String(next));
+    }
+
+    const res = await client.fetch<{
+      deployments: Deployment[];
+      pagination: PaginationOptions;
+    }>(`/v6/deployments?${query}`);
+    deployments = deployments.concat(res.deployments);
+    next = res.pagination.next;
```

Comment:
Do we need to sleep() here in this loop to avoid rate limiting?

Reply:
I think this should be done in the `Client` class. I'll take a look into implementing.

Reply:
Added `sleep(100)`